src/ui/public/styles/variables/color/default.less

```diff
@@ -0,0 +1,589 @@
+//== Colors
+//
+//## Gray and brand colors for use across Bootstrap.
+
+@white:                     #ffffff;
+@black:                     #000000;
+
+@gray2:                     #222222;
+@gray3:                     #333333;
+@gray4:                     #444444;
+@gray5:                     #555555;
+@gray6:                     #666666;
+@gray7:                     #777777;
+@gray8:                     #888888;
+@gray9:                     #999999;
+@gray10:                    #aaaaaa;
+@gray11:                    #bbbbbb;
+@gray12:                    #cccccc;
+@gray13:                    #dddddd;
+
+@gray-base:                 @black;
+@gray-darker:               @gray2;
+@gray-dark:                 #7b8a8b;
+@gray:                      #95a5a6;
+@gray-light:                #b4bcc2;
+@gray-lighter:              #ecf0f1;
+
+@brand-primary:             @gray4;
+@brand-success:             #31C471;
+@brand-info:                #1F6B7A;
+@brand-warning:             #F39C12;
+@brand-danger:              #E74C3C;
+
```

Comment:
So I think we should have a separate file for the color definitions themselves, so it's not duplicated here and in dark-theme.less.